source/views/guides/configuring-diesel-cli.html.slim

```diff
@@ -0,0 +1,119 @@
+section.banner
+  .content-wrapper
+    .banner-wrapper
+      h2.banner__heading Configuring Diesel CLI
+
+main
+  section.demo
+    .content-wrapper
+      .guide-wrapper
+        markdown:
+          Diesel CLI is an optional tool Diesel provides to manage your
+          database schema. It's main two roles are to run database
+          migrations, and create a Rust file which represents your
+          database schema.
+
+          The behavior of Diesel CLI can be configured through a toml file. By
+          default Diesel will look for `diesel.toml` in the same directory as
+          your `Cargo.toml` file. You can provide a different config file by
+          setting the `DIESEL_CONFIG_FILE` environment variable, or passing
+          `--config-file` on the command line. You can get a basic config file
+          with some defaults provided by running `diesel setup`.
```

Comment:
What will `diesel setup` do on a pre-existing project?

Reply:
- Create the migrations directory if it doesn't already exist
- Generate the `000000` migration if it doesn't already exist and PG is being used
- Generate `diesel.toml` if it doesn't already exist
- `diesel database setup`
  - Create the database if it doesn't exist
  - Run migrations

It's designed to be idempotent

Reply:
That's good, do you think it's worth adding a note here that it won't overwrite existing files?

Reply:
Seems reasonable